notification-rule-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_rule.proto

```diff
@@ -16,6 +16,13 @@ message NotificationRuleMutableData {
   string event_condition_id = 3;
   string event_condition_type = 4;
 
-  string channel_id = 5;
+  optional string channel_id = 5;
   string rate_limit_interval_duration = 6;
+
+  NotificationIntegrationTarget integration_target = 7;
+}
+
+message NotificationIntegrationTarget {
+  string type = 1;
```

Comment:
I am not fully onboard with this - still feel enum would be better for a single source of truth. But given it's a small nuance in the scheme of things and it works for our use-case, so works for me.